levels/menu/leveldata.c

```diff
@@ -1933,6 +1933,19 @@ const Gfx dl_menu_texture_kurs_upper[] = {
     gsSPEndDisplayList(),
 };
 #endif
+
+#ifdef ENABLE_SPANISH
+// 0x0700DDA0 - 0x0700EDA0
```

Comment:
this isn't a real address, the other comments are decomp labeling. you can remove this and the Nivel one

Reply:
@Reonu you removed it from German lmao

Reply:
no idea what you're talking about